sdk/python/feast/type_map.py

```diff
@@ -38,6 +39,44 @@
 from feast.value_type import ValueType
 
 
+def feast_value_type_to_python_type(field_value_proto: ProtoValue) -> Any:
+    """
+    Converts field value Proto to Dict and returns each field's Feast Value Type value
+    in their respective Python value.
+
+    Args:
+        field_value_proto: Field value Proto
+
+    Returns:
+        Python native type based on Feast Value Type
```

Comment:
Clarify the docs: 
Python native type representation/version of the given `field_value_proto`